helm/templates/raw-spans-grouper-config.yaml

```diff
@@ -8,12 +8,15 @@ data:
   application.conf: |-
     flink.job {
       {{- if hasKey .Values.rawSpansGrouperConfig.flink "job" }}
+      parallelism = {{ int .Values.rawSpansGrouperConfig.flink.job.parallelism }}
```

Comment:
@ravisingal Should we make this optional instead? Flink defaults to the no. of processors available I think.

Reply:
done.